context.go

```diff
@@ -99,7 +118,7 @@ func (context *Context) run(obj *wafObject, timeout time.Duration, cgoRefs *cgoR
 	result := new(wafResult)
 	defer wafLib.wafResultFree(result)
 
-	ret := wafLib.wafRun(context.cContext, obj, result, uint64(timeout/time.Microsecond))
+	ret := wafLib.wafRun(context.cContext, persistentData, ephemeralData, result, uint64(timeout/time.Microsecond))
```

Comment:
You have to change `waf_dl_unsupported.go` to fix the error on the windows runners: `too many arguments in call to wafLib.wafRun`